ml-agents/mlagents/trainers/tensorflow_to_barracuda.py

```diff
@@ -1550,7 +1548,7 @@ def convert(
 
     # Convert
     o_model = barracuda.Model()
-    o_model.layers, o_input_shapes, o_model.tensors, o_model.memories, o_model.globals = process_model(
+    o_model.layers, o_input_shapes, o_model.tensors, o_model.memories, o_model.globals, = process_model(
```

Comment:
Feel like the trailing comma for these is odd, but 🤷‍♂

Reply:
Oops. I manually reverted some of the changes that a new version of black added, but missed those. Cleaning up now.